index.html

```diff
@@ -121,6 +121,36 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+                <!-- ================================================ -->
+        <!-- ==================  TEMPLATE  ================== -->
+        <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">compudefierro</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/your_user_handle" target="_blank">Your handle</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/your_user_handle" target="_blank">Your handle</a>
```

Comment:
Also note that you need to provide the URL to your profile landing page inside the `href` attribute of `<a>` tag. An example has been given to help you modify :)
```
            <i class="fab fa-github"></i>
            <a href="https://www.github.com/compudefierro" target="_blank">compudefierro</a>
```